host_applications/linux/apps/gencmd/gencmd.c

```diff
@@ -36,6 +36,20 @@ SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
 
 #include "interface/vmcs_host/vc_vchi_gencmd.h"
 
+void show_usage()
+{
+   printf( "Usage: vcgencmd [-t] command\n" );
+   printf( "Send a command to the VideoCore and print the result.\n\n" );
+   printf( "  -t         Time how long the command takes to complete\n" );
+   printf( "Use the command `vcgencmd commands` to get a list of available commands\n\n" );
```

Comment:
Would single-quotes be more appropriate here than backticks? :man_shrugging:

Reply:
That was based what vcdbg outputs when it prints its usage. I'm open to suggestion as to which one is correct.